ibrahim/styles.css

```
@@ -0,0 +1,270 @@
+* {
+  box-sizing: border-box;
+}
+
+/* Undoing defaults */
+body {
+  margin: 0 0 0 0;
```

If all the values are the same, go with this simplified version: 

```suggestion
  margin: 0;
```